source/app.js

```diff
@@ -72,6 +73,11 @@ export default class App extends React.Component {
     if (currentScreen !== prevScreen) {
       tracker.trackScreenView(currentScreen)
     }
+
+    bugsnag.leaveBreadcrumb(`Navigated from to ${currentScreen}.`, {
```

Comment:
One nitpick: "navigation from to" ... having both "from" and "to" is tripping me up.

Reply:
Oops, I left that in. I'm just logging the current screen now.